modules/network-observability-operator-install.adoc

```diff
@@ -9,9 +9,10 @@ You can install the Network Observability Operator using the {product-title} web
 
 .Prerequisites
 
-* Installed Loki. It is recommended to install Loki using the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7].
-* One of the following supported architectures is required: `amd64`, `ppc64le`, `arm64`, or `s390x`.
-* Any CPU supported by Red Hat Enterprise Linux (RHEL) 9
+* If you choose to use Loki, install the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7+].
+* One of the following supported architectures is required: amd64, ppc64le, arm64, or s390x.
+* Any CPU supported by Red Hat Enterprise Linux (RHEL) 9.
+* Must have a cluster installed on infrastructure configured with OVN-Kubernetes, OpenShift SDN, Multus, or SR-IOV as the CNI provider.
```

Comment:
OVNK or OSDN
for customers interested in secondary interface support with SRIOV those customers will additionally need to deploy SRIOV operator

Reply:
@msherif1234 Just want to make sure I'm getting this feedback correct. Are these two things here? 

1. Instead of OVN-Kubernetes, OpenShift SDN, I should have OVN-K, OSDN? 
3. And then, I need to add that if you use SR-IOV as the CNI provider, you will also need to install the SR-IOV Operator.

Reply:
I think it's more about the fact that multus / SR-IOV don't come _alone_, there still need to be ovn-k or osdn (that's not about the naming).
So, what about something like:
```
* Must have a cluster installed on infrastructure configured with OVN-Kubernetes, OpenShift SDN, and optionally using secondary interfaces with Multus and SR-IOV
```
?

Or as an alternative to saying `installed on infrastructure configured with ...` what about:
```
* Must be configured with OVN-Kubernetes or OpenShift SDN as the main network provider, and optionally using secondary interfaces with Multus and SR-IOV
```

Reply:
(afaik "network provider" is the approved term for CNI , unless it changed recently)

Reply:
Sorry, I didn't get to this yesterday. It took me a while to merge all those 1.4 PRs 😅 

I'm good with the 2nd suggestion. And yea...I see OVN-K and SDN are network plugin, and SR-IOV is referred to as a CNI plugin...I guess, we'll just say network plugin and leave it at that.